.gitignore

```diff
@@ -26,6 +26,7 @@ testbin/*
 build/bin
 build/release/
 apiserver.local*
+vendor/
```

Comment:
Remove this line

Reply:
if this repo does not need vendor/ folder, why don't we add the folder to gitignore file?